README.md

```diff
@@ -1,2 +1,18 @@
-# training-operator
-Kubeflow Training Operator
+# Training Operator
+
+## Overview
+
+This repository hosts the Kubernetes Training Operator for Kubeflow training jobs.
+
+## Description
+
+The [Kubeflow Training Operator][1] provides Kubernetes custom resources to run distributed or non-distributed training through so called training jobs, such as TFJobs and PytorchJobs. The Training Operator in this repository is a Python script that wraps the latest released Kubeflow Training Operator [manifests][2], providing lifecycle management and handling events (install, upgrade, integrate, remove). It is part of the [Charmed Kubeflow][3] operators.
```

Comment:
shouldn't it be `one of the  [Charmed Kubeflow][3] operators.`?

Reply:
+1